tests/unit/views/notifications/project_crew_notification.feature

```diff
@@ -145,3 +145,23 @@ Feature: Project Crew Notification
       | Ridcully | usher           | You resigned from the crew of Ankh-Morpork 2010                       |
       | Vetinari | usher           | Mustrum Ridcully resigned from the crew of Ankh-Morpork 2010          |
       | Vimes    | usher           | Mustrum Ridcully resigned from the crew of Ankh-Morpork 2010          |
+
+  Scenario Outline: Ridcully adds themself
```

Comment:
This section should be along with the other add section and not at the bottom of the file.